Src/ReleaseHistory.md

```diff
@@ -18,6 +18,8 @@
 
 ## *Unreleased*
```

Comment:
>Unreleased [](http://example.com/codeflow?start=4&length=10)

dotnet format task and build task is in failed status

Reply:
I have another PR open to fix that. It's caused by an ADO update.